PhysicsTools/PatAlgos/python/slimming/slimmedOOTPhotons_cff.py

```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.PatAlgos.slimming.slimmedPhotons_cfi import *
+
+slimmedOOTPhotons = slimmedPhotons.clone()
+slimmedOOTPhotons.src = cms.InputTag("selectedPatOOTPhotons")
+
+slimmedOOTPhotons.linkToPackedPFCandidates = cms.bool(False)
+slimmedOOTPhotons.recoToPFMap = cms.InputTag("")
+slimmedOOTPhotons.packedPFCandidates = cms.InputTag("")
+slimmedOOTPhotons.saveNonZSClusterShapes = cms.string("(r9()>0.8)") # save additional user floats: (sigmaIetaIeta,sigmaIphiIphi,sigmaIetaIphi,r9,e1x5_over_e5x5)_NoZS 
+#slimmedOOTPhotons.saveNonZSClusterShapes = cms.string("(r9()>0.8 || hcalTowerSumEtConeDR04()<20 || hcalTowerSumEtConeDR04()<0.3*pt())") # save additional user floats: (sigmaIetaIeta,sigmaIphiIphi,sigmaIetaIphi,r9,e1x5_over_e5x5)_NoZS 
```

Comment:
The commented code is not needed; I will drop it.  It was a holdover from when we were discussing the need for HoverE slimming, which as we have shown is not really necessary and will be problematic for the legacy reprocessing.